transports/quic/Cargo.toml

```
@@ -0,0 +1,40 @@
+[package]
+name = "libp2p-quic"
+version = "0.7.0-alpha"
+authors = ["David Craven <david@craven.ch>", "Parity Technologies <admin@parity.io>"]
+edition = "2021"
+description = "TLS based QUIC transport implementation for libp2p"
+repository = "https://github.com/libp2p/rust-libp2p"
+license = "MIT"
+
+[dependencies]
+async-std-crate = { package = "async-std", version = "1.12.0", default-features = false, optional = true }
+async-trait = "0.1.50"
+bytes = "1.2.1"
+futures = "0.3.15"
+futures-timer = "3.0.2"
+if-watch = "2.0.0"
+libp2p-core = { version = "0.37.0", path = "../../core" }
+log = "0.4"
+parking_lot = "0.12.0"
+quinn-proto = { version = "0.8.2", default-features = false, features = ["tls-rustls"] }
+rand = "0.8.5"
+rcgen = "0.9.2"
+ring = "0.16.20"
+rustls = { version = "0.20.2", default-features = false, features = ["dangerous_configuration"] }
+thiserror = "1.0.26"
+tokio-crate = { package = "tokio", version = "1.21.1", default-features = false, features = ["net", "rt"], optional = true }
+webpki = "0.22.0"
+x509-parser = "0.13.0"
+yasna = "0.5.0"
+
+[features]
+tokio = ["tokio-crate"]
+async-std = ["async-std-crate"]
```

Use `dep:` syntax to hide the features created by optional crates.

```suggestion
tokio = ["dep:tokio-crate"]
async-std = ["dep:async-std-crate"]
```